lib/activities/output.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+var fs = require("fs"),
+    path = require("path"),
+    url = require("url");
+
+var debug = require("debug"),
+    mkdirp = require('mkdirp');
+
+var s3Output = require("./s3_output");
+
+var log = debug("swfr:upload");
+
+module.exports = function(outputUri, done, fn) {
+  if(outputUri.indexOf("s3://") === 0) {
+    return s3Output(outputUri, done, fn);
```

Comment:
How do you mean? The return on the method? They both return `outputUri` as far as I can tell.